### ISSUE TITLE
Date and time not displayed

### ISSUE THREAD

**Issue by ipnetxyz** [jd2476/WhatsApp-Chat-HTML-Converter#1]
Thanks for putting together this script, really helpful!

The only issue I had is that for me the date and time is not displayed when using the original code.

In the Whatsapp text file I have the time and date in US style format
[mm/dd/yy, HH:MM:SS AM] or [mm/dd/yy, HH:MM:SS PM]

To make it work, I had to change the following line:
```
timestamp = datetime.strptime(timestamp_str[1:], '%d/%m/%Y, %H:%M:%S')
```
to
```
timestamp = datetime.strptime(timestamp_str[1:], '%m/%d/%y, %H:%M:%S %p')
```
I'm sure there is a more elegant way to do it :)

**Comment by jd2476** [comment]
Wow thank you.  Sorry for the late reply!  I'm in Europe so the format used here is different to the US.

I'll dig into this in a couple of weeks and try to have the script support multiple date formats.

Thanks again!

**Comment by jayb-g** [comment]
Fixed by #2 with many other improvements

**Comment by jd2476** [comment]
Issue fixed
